introspection-engine/introspection-engine-tests/tests/rpc_calls/list_databases_command_tests.rs

```diff
@@ -2,7 +2,7 @@ use barrel::types;
 use introspection_engine_tests::{test_api::*, BarrelMigrationExecutor};
 use test_macros::test_each_connector;
 
-#[test_each_connector(tags("mysql"))]
+#[test_each_connector(tags("mysql"), ignore("vitess"))]
```

Comment:
The reason for this is tracked on Vitess side at https://github.com/vitessio/vitess/issues/7687